libear/CMakeLists.txt

```diff
@@ -31,7 +31,9 @@ if(CMAKE_THREAD_LIBS_INIT)
 endif()
 
 set(CMAKE_MACOSX_RPATH 1)
-set_target_properties(ear PROPERTIES INSTALL_RPATH "@loader_path/${CMAKE_INSTALL_LIBDIR}")
+if(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
```

Comment:
I'm wondering is this the correct platform check? In the `config.h.in` there is a variable called `APPLE` would it be better to reuse that?